### PR TITLE
Certificate download makeover

### PR DESCRIPTION
**Pull request by marians**
RFR @puja108 @oponder 

To be merged into `secondgen`

### CODE REVIEW

File: content/guides/accessing-services-from-the-outside/index.md [hunk 1]
```diff
@@ -20,17 +20,25 @@ The Kubernetes API comes with an inbuilt proxy, which you can use to access serv
 
 ### Get Certificates
 
-First, you need the certificates in the right format.
+First, you need the certificates and key in the right format. You can generate and download them from within our [web user interface](https://happa.giantswarm.io/docs/configure). There, after kreating a new key pair, you get the option to download three files needed:
```
Comment (puja108) [inline content/guides/accessing-services-from-the-outside/index.md]: kreating->creating
download three files needed: -> download the three needed files

File: content/guides/accessing-services-from-the-outside/index.md [hunk 2]
```diff
@@ -20,17 +20,25 @@ The Kubernetes API comes with an inbuilt proxy, which you can use to access serv
...
 
-LINK TO HAPPA [ca.crt]() [kubecfg.key]() [kubecfg.crt]()
+- the certificate file for the CA (certificate authority) of your cluster, named `ca.crt`
+- the certificate of your key-pair, named `client.crt`
```
Comment (puja108) [inline content/guides/accessing-services-from-the-outside/index.md]: key-pair -> key pair
(or change the below to include `-`)

File: content/guides/accessing-services-from-the-outside/index.md [hunk 3]
```diff
@@ -20,17 +20,25 @@ The Kubernetes API comes with an inbuilt proxy, which you can use to access serv
...
+- the private key of your key pair, named `client.key`
 
-You, then need to create a `PKCS#12` file from your public key and cert pair:
+Once you have downloaded these, you'll have to convert the client key and certificate into a format usable by the web browser (or your operating system's key chain), which is the PKCS#12 format. This OpenSSL command will do this for you:
```
Comment (puja108) [inline content/guides/accessing-services-from-the-outside/index.md]: key chain -> keychain

**Comment by marians** [comment]
@puja108 Edited. Thanks!

**Pull request merged**